engine/access/rpc/backend/script_executor.go

```diff
@@ -21,25 +22,41 @@ type ScriptExecutor struct {
 	// initialized is used to signal that the index and executor are ready
 	initialized *atomic.Bool
 
-	// init is used to ensure that the object is initialized only once
-	init sync.Once
+	// minCompatibleHeight and maxCompatibleHeight are used to limit the block range that can be queried using local execution
+	// to ensure only blocks that are compatible with the node's current software version are allowed.
+	// Note: this is a temporary solution for cadence/fvm upgrades while version beacon support is added
+	minCompatibleHeight *atomic.Uint64
+	maxCompatibleHeight *atomic.Uint64
```

Comment:
are they usually configured together? Do we consider using `compabitleHeightRange`?

Reply:
no, generally only one would be defined. both are only defined for ANs running a version that's older than the HCU version.

FWIW, we'll be able to remove these once the version beacon support is added